docs/corpora_schema.md

```diff
@@ -69,44 +84,44 @@ corpora_encoding_version|Follows [SemVer](https://semver.org/) conventions
+In addition to these free text fields (except sex), each cell MUST also have ontology values:
 
 **Field name**|**Constraints**
 :--|:--
-tissue\_ontology\_term\_id|UBERON term
+tissue\_ontology\_term\_id|UBERON term. This field SHOULD be appended with " (cell culture)" or " (organoid)" if appropriate.
 assay\_ontology\_term\_id|EFO term
 disease\_ontology\_term\_id|MONDO term or [PATO:0000461](http://bioportal.bioontology.org/ontologies/PATO?p=classes&conceptid=PATO%3A0000461)
 cell\_type\_ontology\_term\_id|CL term
 ethnicity\_ontology\_term\_id|HANCESTRO term, "na" if non-human
 development\_stage\_ontology\_term\_id|HsapDv term if human, child of EFO:0000399 otherwise
 
-The `tissue_ontology_term_id` field must be appended with " (cell culture)" or " (organoid)" if appropriate.
+cellxgene requires ontology terms to enable search, comparison, and integration of data. When no appropriate ontology value is available, then the most
+precise accurate term MUST be used. For example if the `cell_type` field describes a relay interneuron, but the most specific available term in the CL
+ontology is CL:0000099 ("Interneuron"), then the interneuron term can be used to fulfill this requirement, and ensures that users searching for "neuron"
+are able to find these data. Users will still be able to access more specific cell type annotations that have been submitted with the
+data (but aren't required by the schema).
 
-The `ontology_term_id` fields may be empty strings when no appropriate ontology value is available, for example if the
-`cell_type` field describes a cell type for which no ontology term exists or if the `ethnicity` is unknown. If the
-field is not empty, then it must be an [OBO-format ID](http://www.obofoundry.org/id-policy.html), meaning it is a CURIE
+Ontology terms MUST use [OBO-format ID](http://www.obofoundry.org/id-policy.html), meaning they are a CURIE
 where the prefix identifies the ontology. For example `EFO:0000001` is a term in the `EFO` ontology.
 
-If the features of the dataset are human genes, then the feature ids must be [HGNC](https://www.genenames.org/about/guidelines/#!/#tocAnchor-1-7) approved
+If the features of the dataset are human genes, then the feature ids MUST be [HGNC](https://www.genenames.org/about/guidelines/#!/#tocAnchor-1-7) approved
 gene symbols.
 
-Similarly if the features are mouse genes, then the feature ids must be [MGI](http://www.informatics.jax.org/mgihome/nomen/gene.shtml) gene symbols.
+Similarly if the features are mouse genes, then the feature ids SHOULD be [MGI](http://www.informatics.jax.org/mgihome/nomen/gene.shtml) gene symbols.
 
-Finally, the whole dataset must be annotated with fields that indicate the organism and describe the meanings of the [matrix layers](#Matrix-Layers):
+Finally, the whole dataset MUST be annotated with fields that indicate the organism and describe the meanings of the [matrix layers](#Matrix-Layers):
```

Comment:
```suggestion
The whole dataset MUST be annotated with fields that indicate the organism and describe the meanings of the [matrix layers](#Matrix-Layers).
```

Reply:
Should I split this sentence in two and move the matrix layers portion to the layers section? Could we be more specific about how this is implemented? What is the code string? (similar to e.g. `AnnData.layers["raw"]`)

Reply:
See 60378ec